### ISSUE TITLE
Whonix sdwdate-gui tray widget disappears when compositing is disabled

### ISSUE THREAD

**Issue by UndeadDevel** [QubesOS/qubes-issues#8797]
### Qubes OS release
4.2, Whonix 17
(also happened on 4.1.2 with Whonix 16)

### Brief summary
The `sdwdate-gui` tray widget disappears from the tray when "Enable display compositing" is turned off; turning compositing back on does not bring it back; launching the `sdwdate-gui` app from the app menu launches a new instance without information about the sdwdate status of `sys-whonix` or any Whonix workstations. Only restarting `sys-whonix` brings it back in the correct configuration.

### Steps to reproduce
1. Launch `sys-whonix`
2. Launch the "Window Manager Tweaks" app and in the "Compositor" tab toggle "Enable display compositing"

### Expected behavior
Toggling display compositing does not affect the sdwdate-gui widget.

### Actual behavior
Removes the widget.

**Comment by adrelanos** [comment]
* affects Qubes-Whonix: yes
* caused by Qubes-Whonix: no, probably not.

sdwdate-gui(-qubes) is just a python application that shows a systray.

Where is this `Enable display compositing`? Qubes dom0 -> window manager tweaks? Enabled by default for me. Not reproducible for me.

> launching the sdwdate-gui app from the app menu launches a new instance without information about the sdwdate status of sys-whonix or any Whonix workstations.

* https://github.com/QubesOS/qubes-issues/issues/8798

**Comment by UndeadDevel** [comment]
>Where is this Enable display compositing? Qubes dom0 -> window manager tweaks? Enabled by default for me. Not reproducible for me.

Yeah, that's the one; for me it's also enabled by default, but when testing some things I disabled it and the taskbar was affected (some graphical glitches briefly occurred) and after that the sdwdate-gui tray widget was gone; re-enabling compositing did not restore it...it's really not a big deal as the vast majority of people will not be switching compositing off and on again, and, as you say, this may not occur on all setups.
I am using a fresh install of 4.2 with a (almost) certified NV41 (NK instead of novacustom) without any major mods (KDE or others), though, so I thought it worthwhile to report.

I'll leave it open for now; maybe someone else can reproduce it and is more impacted by this.

**Comment by DemiMarie** [comment]
Is this worth fixing?

**Comment by UndeadDevel** [comment]
>Is this worth fixing?

Not if it entails a lot of effort would be my take...the use case(s) where this issue would be a significant pain seems very niche. Feel free to close as "won't do" or similar.
